examples/PeriodicOrbitalMCMC.md

```diff
@@ -310,7 +310,11 @@ First we define our parametrized MAF bijection using autoregressive neural netwo
 
 ```{code-cell} ipython3
 import optax
-from numpyro.nn import AutoregressiveNN
+try:
+    from numpyro.nn import AutoregressiveNN
+except ModuleNotFoundError:
+    %pip install -qq numpyro
```

Comment:
We currently does the same in examples/change_of_variable_hmc.md
I guess here we kind of need to do that because otherwise this example does not render, see: https://blackjax-devs.github.io/blackjax/examples/PeriodicOrbitalMCMC.html#ellipsis-iaf

Reply:
The clean solution to this issue is to add Numpyro in `requirements-doc.txt` (which should be moved to this foder btw).

Reply:
Just saw your comment, I will update this PR first then

Reply:
Testing locally now, will push 🔜